### PR TITLE
improved diffviewer to work with wikidata #1347

### PR DESCRIPTION
**Pull request by heysusgarcia**

### CODE REVIEW

File: app/assets/javascripts/components/revisions/diff_viewer.jsx [hunk 1]
```diff
@@ -60,7 +60,16 @@ const DiffViewer = React.createClass({
   },
 
   wikiUrl() {
-    return `https://${this.props.revision.wiki.language}.${this.props.revision.wiki.project}.org`;
+    const wikiProject = this.props.revision.wiki.project;
+    let url;
+
+    if (wikiProject === 'wikipedia') {
```
Comment (ragesoss) [inline app/assets/javascripts/components/revisions/diff_viewer.jsx]: Most projects work the same way as Wikipedia in terms of the url: es.wikisource.org, ta.wiktionary.org, etc. There are just a handful that have a different url pattern; I think the only ones we currently support are wikidata (www.wikidata.org) and multilingual wikisource (wikisource.org).

Comment (heysusgarcia) [reply app/assets/javascripts/components/revisions/diff_viewer.jsx]: That is good to know. As it stands, my logic will only work for wikipedia and wikidata projects. If I reverse my logic as follows: check if wikiProject === 'wikidata' , else return the url pattern we had previously been returning that supports wikipedia, wikitionary, etc url patterns, will that suffice?

Comment (ragesoss) [reply app/assets/javascripts/components/revisions/diff_viewer.jsx]: I think so. You can add some activity from users active on a couple different wikis and try it out.

**Comment by heysusgarcia** [comment]
@ragesoss my branch has been updated with the revised logic.

**Pull request closed without merging**